src/main/java/edu/hm/hafner/analysis/registry/CodeCheckerDescriptor.java

```diff
@@ -0,0 +1,22 @@
+package edu.hm.hafner.analysis.registry;
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.parser.CodeCheckerParser;
+
+/**
+ * A descriptor for the Codechecker parser.
+ *
+ */
+class CodeCheckerDescriptor extends ParserDescriptor {
+    private static final String ID = "CodeChecker";
```

Comment:
Typically IDs are all lowercase and use a dash as separator. They are used in URLs later on.
```suggestion
    private static final String ID = "code-checker";
```